tests/test_pdo.py

```diff
@@ -302,14 +302,15 @@ def test_create_poller(motion_controller):
     skip_if_pdo_padding_is_not_available(mc, alias)
     registers = [{"name": "CL_POS_FBK_VALUE", "axis": 1}, {"name": "CL_VEL_FBK_VALUE", "axis": 1}]
     sampling_time = 0.25
-    sleep_time = 1
+    samples_target = 4
+    sleep_time = (samples_target - 0.5) * sampling_time
```

Comment:
This is the best practice for test stability.
Sleeping a multiple of sampling time could be unstable